### PR TITLE
Update Mob Stats

### PR DESCRIPTION
**Pull request by Xaver-DaRed**
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] that I agree to LandSandBoat's [Limited Contributor License Agreement](https://github.com/LandSandBoat/server/blob/base/.github/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I have **read the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)**
- [x] that I've _**tested my code and things my code changed**_ since the last commit in the PR, and will test after any later commits

First pass at Mob statistics. Several issues arrised when editing the data wich will have to be adressed... With time.
Data taken from:
- https://docs.google.com/spreadsheets/d/1giECv1qkjhXP691e2ioQoOD0EnbYabdSY2eO2Ny-n7Q/edit?usp=sharing
- https://docs.google.com/spreadsheets/d/14sDZbtDw8o3v7oQvMMKKUSdyHdKAqkM47wSiFOHtN6U/edit#gid=1124478195
- https://docs.google.com/spreadsheets/d/1TepERJy1kEHvobxZzRU4sGffmeWK_AYaSV-92LjkwFk/edit#gid=1258860949

Special thanks to Jimmayus, Aether, Meloetta, Juul, Yve, Meyi/Lekoh for all the hard work and to Eden server players and devs.

### CODE REVIEW

**Comment by ShiyoKozuki** [comment]
Monsters are supposed to have A+(1) in attack and accuracy. Yours are set to 3.

**Comment by Xaver-DaRed** [comment]
Atk and Acc have not been updated yet. Just main stats (except HP and MP), Def and Eva.
Those 2, or at least Eva, will need an update at a later date anyway, since it is suposed to be Job dependant.

If they are gonna be 1 always, they should just be removed outright, IMO, but that would require some talk and planning, since the family system in itself needs some love.

Thank you for the info, anyway^^

File: src/map/utils/mobutils.cpp [hunk 1]
```diff
@@ -427,22 +427,22 @@ namespace mobutils
...
+            sSTR /= 4;
+            sDEX /= 4;
+            sAGI /= 4;
+            sINT /= 4;
+            sMND /= 4;
+            sCHR /= 4;
+            sVIT /= 4;
+        }
+
+        PMob->stats.STR = floor(fSTR) + floor(mSTR) + floor(sSTR);
```
Comment (claywar) [inline src/map/utils/mobutils.cpp]: All of these look like they're declared as uint values.  There's no reason to floor

File: src/map/utils/mobutils.cpp [hunk 2]
```diff
@@ -415,7 +415,10 @@ namespace mobutils
         uint16 sMND = GetBaseToRank(grade::GetJobGrade(PMob->GetSJob(), 7), PMob->GetSLevel());
         uint16 sCHR = GetBaseToRank(grade::GetJobGrade(PMob->GetSJob(), 8), PMob->GetSLevel());
 
-        if (PMob->GetSLevel() > 15)
```
Comment (TeoTwawki) [inline src/map/utils/mobutils.cpp:418]: ~~As a heads up to people using modified subjob lv for their monsters: that setting will no longer affect mob stats in game with this change.~~ actually no, I missed a block of code where they still start life form sub, so thats still going to have an effect on them. just the lowest lv mobs will get changed a bit. Everything 45 and up still gets the same math.

However, **this change IS retail accurate** and needs to happen. This of us rolling custom need to just deal with it. :+1:

**Review by TeoTwawki — APPROVED** [review]

**Pull request merged**